docs/src/howtouse/creatingModels.md

```diff
@@ -58,35 +59,32 @@ This fields sets a vector of nodal masses components $[m_x, m_y, m_z]$ that is a
 
-### `elements.elemTypeAero`
+### `elements.massMatType`
+The `massMatType` field sets, for frame or truss elements, whether consistent or lumped mass matrix is used for the inertial term in dynamic analyses. The `massMatType` field should be set as a string variable: `'consistent'` or `'lumped'`,  and if it is not declared then by default the `'lumped'` mass matrix is set.
+ 
+### `elements.elemCrossSecParams`
+This is a cell structure with the information of the geometry of the element.
 
-If a frame aerodynamic analysis is desired, the  `elementTypeAero` field should contain a vector with: in the first three entries: three coordinates of the aerodynamic chord vector (the system of coordinates considered for this is the local reference system at the undeformed configuration), then in the fourth entry: the number of Gauss integration points $numGauss$ and in the fifth entry a boolean $computeAeroBool$ which indicates if the aerodynamic stiffness matrix is computed:
-```math
-[ vch_{t1} \,\, vch_{t2} \,\, vch_{t3} \,\,numGauss\, \,computeAeroBool ]
-```
+### `elements.dragCoefFunction`,   `elements.liftCoefFunction` and `elements.pitchCoefFunction`
+If a frame aerodynamic analysis is desired, the drag, lift and pitch moment should be defined in this field. Each function receives Reynolds and the incidence angle as inputs and returns the the respective coefficient. For some `elemCrossSecParams` like `'circle'` internal built-in functions are set as default.If any of the coefficients is defined considered then an empty `[]` struct is considered, this set this coefficient to zero.
 
-### `elements.aeroCoefs`
+### `elements.chordVector`
+A vector with the three coordinates of the aerodynamic chord vector (the system of coordinates considered for this is the local reference system at the undeformed configuration)
 
-If a frame aerodynamic analysis is desired, the `aeroCoefs` field must contain a column cell with the names of the functions (as strings) considered to compute the aerodynamic forces, as follows:
+### `elements.aeroNumericalParams`
+A cell with the number of Gauss integration points `numGauss`, the boolean `computeStiffnessAeroTangent` for computing the aerodynamic stiffness matrix and `geometricNonLinearAero` to take into account geometric nonlinearities or (reconfiguration).  
 ```math
-\{ `dragCoefFunction`; `liftCoefFunction`; `momentCoefFunction` \}
+\{  numGauss \,\,stiffnessAeroTangent\, \,geometricNonLinearAero \}
 ```
-Each function must recieve as arguments: first the relative angle of incidence and second the Reynolds number. If any of the coefficients is not considered then an empty `[]` can be used.
-
-### `elements.elemCrossSecParams`
-
-This is a cell structure with the information of the geometry of the element.
-
+where the default cell is `{4, false, true}`
 #### 1D elements
```

Comment:
all of these descriptions should go after the `elements.elemCrossSecParams` title...

Reply:
Done